### ISSUE TITLE
No developer drop visible on Binance Web3 wallet

### ISSUE THREAD

**Issue by MaxCCC** [fluencelabs/dev-rewards#145]
Hi, I'm not really familiar with this. After going through the process I claimed the drop. 
I don't see the value in my web3 wallet on binance. When I click on transaction history, then I see I received 2500 FTL-DROP but I can't see the assets somewhere. Does it take 2 months before I can see this (lock up period)?

**Comment by daviddavo** [comment]
Yeah, in two months you have to click "I already have FLT-DROP" here: https://claim.fluence.network
